docs/guides/wifi.md

```diff
@@ -1,120 +1,52 @@
+1. Boot into macOS.
 
-- You can check your Model Identifier like this:
-    - On macOS: `system_profiler SPHardwareDataType | grep "Model Identifier"`
-    - On Linux: `cat /sys/devices/virtual/dmi/id/product_name`
-- If unknown, your Mac's wifi chipset number can be found like this:
-    - On macOS: `ioreg -l | grep RequestedFiles`, the number will be
-      within the folder names, i.e. "C-`4364`\_\_s-B3" (Your model's
-      island will also be in this command's output).
-    - On Linux: `lspci -d '14e4:*'`, this will also tell you the
-      "Rev"/Revision.
+2. Open the terminal and run `cp -r /usr/share/firmware/wifi ~/Desktop`. This shall make a folder named **wifi** on the macOS Desktop.
 
-| Model Identifier | Chipset | Revision | Island   | Firmware Options     |
-|------------------|---------|----------|----------|----------------------|
-| MacBookPro16,1   | BCM4364 | 4        | Bali     | Big Sur              |
-| MacBookPro16,2   | BCM4364 | 4        | Trinidad | Big Sur              |
-| MacBookPro16,3   | BCM4377 | 4        | Tahiti   | Big Sur              |
-| MacBookPro16,4   | BCM4364 | 4        | Borneo   | Big Sur              |
-| MacBookPro15,1   | BCM4364 | 3        | Kauai    | Mojave / Big Sur     |
-| MacBookPro15,2   | BCM4364 | 3        | Maui     | Mojave / Big Sur     |
-| MacBookPro15,3   | BCM4364 | 3        | Kauai    | Mojave / Big Sur     |
-| MacBookPro15,4   | BCM4377 | 4        | Formosa  | Big Sur              |
-| MacBookAir9,1    | BCM4377 | 4        | Fiji     | Big Sur              |
-| MacBookAir8,1    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacBookAir8,2    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacMini8,1       | BCM4364 | 3        | Lanai    | Mojave / Big Sur     |
-| MacPro7,1        | BCM4364 | 4        | Kahana   | Big Sur              |
-| iMac20,1         | BCM4364 | 4        | Hanauma  | Big Sur              |
-| iMac20,2         | BCM4364 | 4        | Kure     | Big Sur              |
-| iMac19,1*        | BCM4364 | 3        | Midway   | Mojave / Big Sur     |
-| iMac19,2*        | BCM4364 | 3        | Nihau    | Mojave / Big Sur     |
-| iMacPro1,1       | BCM4364 | ?        | Ekans?   | Mojave / Big Sur?    |
+3. Copy the **wifi** folder to an external drive or a partition which is formatted to a file system readable by Linux. You can also use your EFI partition by mounting it using `sudo diskutil mount disk0s1`.
 
-\*iMac19,x don't have the T2 chip, but they have the same type of wifi chip as T2 Macs.
+4. Boot into Linux.
 
-*If there is missing/uncertain information for your model, please open a [pull request](https://github.com/t2linux/wiki/edit/master/docs/guides/wifi.md) or [issue](https://github.com/t2linux/wiki/issues/new) or mention it on the [discord](https://discord.com/invite/68MRhQu).*
+5. Copy the **wifi** folder to the **Home** folder using your file manager.
```

Comment:
Could you also mention that if they put the firmware on the eif partition, they can access it with `sudo mount /dev/nvme0n1p1 /mnt`

Reply:
Won't they have to unmount it first from `/boot/efi`(if using it as EFI on Linux too)

Reply:
Good point, they'd have to check /boot/efi, if its not there then they'd have to mount it

Reply:
Exactly. This shall make the guide complicated. We shall have to make 2 cases now. A better option seems to be the one I mentioned in the guide. Also, for advanced users, the line about EFI I mentioned seems appropriate enough for them to get it.